README.md

```diff
@@ -14,11 +16,16 @@ returns on investment. ADRIA uses a set of dynamic Multi-Criteria Decision Analy
 which simulates a reef decision maker to identify candidate locations for intervention
 deployment which consider ecological, economic and social benefits.
 
-ADRIA also includes a simplified coral ecosystem model to allow exploration of outcomes as a 
+ADRIA also includes a simplified coral ecosystem model to allow exploration of outcomes as a
 result of intervention decisions made across a wide range of possible future conditions.
 
 ADRIA requires Julia v1.9 and above.
 
-Setup and usage is demonstrated in the [documentation](https://open-aims.github.io/ADRIA.jl/stable/usage/getting_started/)
+Setup and usage is demonstrated in the
+[Documentation](https://open-aims.github.io/ADRIA.jl/stable/usage/getting_started/).
+
+For developers, refer to the
+[Developers setup guide](https://open-aims.github.io/ADRIA.jl/stable/development/development_setup/).
 
-For developers, refer to the [Developers setup guide](https://open-aims.github.io/ADRIA.jl/stable/development/development_setup/)
+This project is currently moving to use [Blue Style Guide](https://github.com/invenia/BlueStyle).
+Please, try to follow the style guide when submiting a PR.
```

Comment:
Move these statements above or next to the badge.

Remove "please" and "try".
https://docs.openstack.org/doc-contrib-guide/writing-style/general-writing-guidelines.html#eliminate-needless-politeness
https://developers.google.com/style/word-list#please

Reply:
@ConnectedSystems done. Just one note: moving these statements above the badge make them the first thing you read when you enter ADRIA's README. That's why I had put them at the end.

Reply:
Okay, below? Currently the statement is too far from the badge so it's a little deceptive. Alternatively, move the badge down until at least most of the code has been updated.

Reply:
When I try to put them at the same line using Markdown only, it get's vertically unaligned

![image](https://github.com/open-AIMS/ADRIA.jl/assets/8040719/22c23884-da3b-4494-8486-3690e63795e6)

And when I jump one line between them, the distance between the badge and the next paragraph is bigger than to the previous:

![image](https://github.com/open-AIMS/ADRIA.jl/assets/8040719/8aa2ce7b-2921-4126-b8c0-8d9820bd3869)


For that reason I have used HTML.

Reply:
Yep, that's fine.